frigate/stats.py

```diff
@@ -13,11 +15,25 @@
 logger = logging.getLogger(__name__)
 
 
+def get_latest_version() -> str:
+    try:
+        raw = urlopen('https://api.github.com/repos/blakeblackshear/frigate/releases/latest')
```

Comment:
I have included the `requests` library as a part of integrating Frigate+. We should use that here too.

Reply:
Ah nice, that'll be cleaner